src/oca_github_bot/config.py

```diff
@@ -53,3 +53,6 @@ def func_wrapper(*args, **kwargs):
 #  delete_branch,tag_approved,tag_ready_to_merge,gen_addons_table,gen_addons_readme,
 #  gen_addons_icon,setuptools_odoo,merge_bot
 BOT_TASKS = os.environ.get("BOT_TASKS", "all").split(",")
+
+GITHUB_STATUS_IGNORED = ["ci/runbot", "codecov/project", "codecov/patch"]
```

Comment:
I wouldn't ignore ci/runbot, as it can be red due to legit reasons.

Reply:
I'm not sure, runbot tend to be red for technical issues more often than not these days. 
The person requesting the merge is supposed to check everything is okay first. This additional test run is only to check issues after rebase so I think it's better to have it relatively lightweight and as reliable as possible.

Reply:
@sbidoul I still think we should add this check, as now the merge command serves as "merge this when you finish", and we don't wait for tests results, and maybe a runbot warning can find its way to the main branch.